### PR TITLE
Run HIL tests on Hedwig

### PR DESCRIPTION
**Pull request by OYTIS**
Under test yet.

### CODE REVIEW

**Comment by OYTIS** [comment]
Looks ready now.

**Comment by OYTIS** [comment]
OK, now it only measures coverage of relevant files, that's cool.

File: .travis.yml [hunk 1]
```diff
@@ -5,9 +5,9 @@ compiler:
...
+  - secure: efbSNAbENJit6C6PDnmvsAMNmMD9+jLYHE2WsoutzVhMz/HUgNLfRC+Y0gAlVEOVhUpRh3I+5ZMqHt38zkx5JxdlQpR7J2dZQiVKuQL3Drqa9DWMQFmUtwvCQMyG2SS2z2433MfxKrbr3GNpTv9f3ee8fiOnGMYZytCnAiu6V6mgvcwi+5PGhT3yPUtagQxyOJvxqI2mPNe97Uasj5DcFQ2mFkXeveJrxET0fij4F/xJNaDDyvtCeKMNCym7dyl47Tfqj3qK07Mchf1l0GFFno4IlavlzgkXu6UQvnitcc9NN2KCjFZDQpX1e8pxuqEnEP3vSB8PVQummFEJfNge/nGiBFdVwAt4S+hH9jiwSsLxM+KHbOI/sKJn/vWZlOMbZEMQlgdv0vbpvJn3WE3zrOTNUgUUgKYUqkICvv+tSzJr+SgprbQ8A+wxXbMuXSYnVoJVBcxWCmT2IfbxVQZ0iFV3K630bNaFL090JRhnzGKwh/dVbRIlgv5QBP9IlCU4DnCfMGk4XwuTHT20rd7JWCrL2nU10CtZy/w8U01JCRR+9KuHL5+XPmsHrWfXRK/KYbWcNqwjVd8Ae38Wix6DKSbYLvfX1nHD02dMZVPiYwVkWqLajFXkN8CjsqX8jyCcqNwUa2lUT1nERBraqM+4GOp6llDAFTmof6YApnvdUXk=
```
Comment (rbeiter) [inline .travis.yml]: Is what these properties are (property name and purpose) and how they are converted to the secure string for travis mentioned in Hedwig's README?

**Review by rbeiter — APPROVED** [review]
Pending Q on travis secure env properties, lgtm!

File: .travis.yml (hunk 1 above)
Comment (OYTIS) [reply .travis.yml]: One should be `SECRET_HEDWIG_WORKERS=...` as described in https://github.com/twilio/breakout-massive-iot/pull/28/files#diff-4d6784c0365d7ba6724dfc594c86489fR127 (this one includes `HEDWIG_AUTH_TOKEN`). Another one is `CODECOV_TOKEN=...`

**Pull request merged**